### ISSUE TITLE
Accessibility: element using 'aria-activedescendant' is not tabbable

### ISSUE THREAD

**Issue by makandre** [carbon-design-system/carbon-components-angular#2368]
## Detailed description

> Describe in detail the issue you're having.

The dropdown component has a new violation showing up.  I believe this is because of a recent update of the rule in IBM Equal Access Accessibility Checker, via https://github.com/IBMa/equal-access/pull/1099

```
The <ul> element using 'aria-activedescendant' set to "listbox-0-4" is not tabbable
 Violation
Element using 'aria-activedescendant' property should be tabbable
```

Interestingly, `ibm-combo-box` which is very similar to the dropdown, renders its listbox without the `aria-activedescendant` attribute, and hence does not have this violation.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

`ibm-dropdown`

> What did you expect to happen? What happened instead? What would you like to see changed?

There should be no accessibility violation

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

`"carbon-components-angular": "^4.63.0"`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM ADDI

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

* Screenshots or code

<img width="801" alt="image" src="https://user-images.githubusercontent.com/17596240/206166240-4fdbc921-1abb-4443-882d-ca4a38a5fc93.png">

* Notes
